### PR TITLE
[new release] libdash (v0.3)

### PR DESCRIPTION
**Pull request by mgree**
Improvements to pretty printer; new `json_to_shell` and `shell_to_json` executables.

Libdash also now has [Python bindings](https://pypi.org/project/libdash/).

### CODE REVIEW

File: packages/libdash/libdash.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Bindings to the dash shell's parser"
+maintainer: "Michael Greenberg <michael.greenberg@pomona.edu>"
+authors: "Michael Greenberg <michael.greenberg@pomona.edu>"
+license: "BSD-3-Clause"
+homepage: "https://github.com/mgree/libdash"
+bug-reports: "https://github.com/mgree/libdash/issues"
+depends: [
+  "ocaml" {>= "4.0.7"}
```
Comment (kit-ty-kate) [inline packages/libdash/libdash.0.3/opam:9]: Are you sure you mean 4.0.7 ? There is no such version of the compiler. Maybe you meant 4.07
```suggestion
  "ocaml" {>= "4.07"}
```

File: packages/libdash/libdash.0.3/opam [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  "ocamlfind" {>= "1.8.0"}
+  "ctypes" {>= "0.18.0"}
+  "ctypes-foreign" {>= "0.18.0"}
+  "atdgen" {>= "2.3.2"}
+  "opam-installer" {>= "2.1.0"}
```
Comment (kit-ty-kate) [inline packages/libdash/libdash.0.3/opam:14]: This should not be required
```suggestion
```

File: packages/libdash/libdash.0.3/opam [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  [make "install"] # into _build
+  ["ocaml/mk_meta.sh" "%{_:lib}%"] # pass along the lib directory for the rpath in the META
+  [make "-C" "ocaml" "all"]
+  ["./mk_dot_install.sh"]
+  ["./ldconfig.sh"] # fix up .so files if ldconfig didn't do it
+  [make "-C" "ocaml" "test"] {with-test}
+]
+install: [
+  ["opam-installer" "--prefix=%{prefix}%" "libdash.install"]
+]
```
Comment (kit-ty-kate) [inline packages/libdash/libdash.0.3/opam:38]: opam does this automatically already
```suggestion
```

**Comment by mgree** [comment]
Ha, good catch on the compiler version. Looks good to me! Thanks again, as always. 😁

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**